src/Doctrine/Odm/PartialPaginator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Doctrine\Odm;
+
+use ApiPlatform\Metadata\Exception\RuntimeException;
+use Doctrine\ODM\MongoDB\Iterator\Iterator;
+use Doctrine\ODM\MongoDB\UnitOfWork;
+
+final class PartialPaginator extends AbstractPaginator
+{
+    private readonly \ArrayIterator $iterator;
+    private readonly int $count;
```

Comment:
This property should be declared in `AbstractPaginator`, as it's used in `getCount`.